### PR TITLE
All the Emojis

### PR DESCRIPTION
**Pull request by githubstudent**
All words that possibly line up with emoji shapes, let's switch from characters to emoji.

### CODE REVIEW

**Comment by githubstudent** [comment]
@jordanmccullough, Would you review the changes to the words that replace characters with emoji?

**Comment by jordanmccullough** [comment]
Looks like we might have a few more words to switch out.

**Comment by githubstudent** [comment]
Works for me!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # NICAR_2016
 Simple Repository
 
-Time to spruce this puppy up! :coffee:
```
Comment (jordanmccullough) [inline README.md:4]: "spruce" could be replaced with `:evergreen:`

Comment (githubstudent) [inline README.md:4]: I'll think about it.

**Comment by githubstudent** [comment]
Let's go for it sans :evergreen_tree:.

**Pull request merged**